### PR TITLE
Update @ckeditor/ckeditor5-widget to v32

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
https://github.com/ckeditor/ckeditor5/blob/9f6e48af6b7c819506258d10df8d0f26a952996d/packages/ckeditor5-widget/src/utils.js#L170-L171
https://github.com/ckeditor/ckeditor5/commit/a1ae56110e24165b14e0a84b696500ea05fb9e25#diff-8dfba86084e963acb7c47f7f972c47897b262344f9911a532b2a05e36f61ae1eL399
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ckeditor__ckeditor5-widget ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ckeditor__ckeditor5-widget@latest/))
was missing the following properties:
1. centeredBalloonPositionForLongWidgets
1. centeredBalloonPositionForLongWidgets
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 32262f511c7a580e6c241933a33cbda9a7f50e1c
</p>

**Comment by fedemp** [comment]
> Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files. The check for missing properties isn't always right, so take this list as advice, not a requirement.
> ## ckeditor__ckeditor5-widget ([unpkg](https://unpkg.com/browse/ckeditor__ckeditor5-widget@latest/))
> 
> was missing the following properties:
> 
>     1. centeredBalloonPositionForLongWidgets
> 
>     2. centeredBalloonPositionForLongWidgets
> 
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [32262f5](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/32262f511c7a580e6c241933a33cbda9a7f50e1c)

Fixed. I forgot to update the header in `index.d.ts`

**Review by martin-badin — CHANGES_REQUESTED** [review]
Please update the version as well. Sorry, I see

**Review by martin-badin — APPROVED** [review]

File: types/ckeditor__ckeditor5-widget/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for @ckeditor/ckeditor5-widget 29.0
+// Type definitions for @ckeditor/ckeditor5-widget 32.0
```
Comment (martin-badin) [inline types/ckeditor__ckeditor5-widget/index.d.ts:1]: the latest version is 33

**Review by martin-badin — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**